### PR TITLE
[Issues #48] Fix indentation and bg in highlight code block

### PR DESCRIPTION
**Pull request by saikocat**
Refer to issues #48 

Here are the images detailing the issue: https://imgur.com/a/ysqlotN

### CODE REVIEW

**Comment by alex-shpak** [comment]
Hi!
Thank you for contribution.

File: assets/_markdown.scss [hunk 1]
```diff
@@ -67,4 +67,11 @@ $block-border-radius: 0.15rem;
       background: $gray-100;
     }
   }
-}
\ No newline at end of file
+
+  .highlight {
```
Comment (alex-shpak) [inline assets/_markdown.scss]: If I remember correct it should be rather `pre` instead of `.highlight`

Comment (alex-shpak) [reply assets/_markdown.scss]: You can put 
```
  code {
      padding: 0;
      background: none;
    }
```
Into line 45 I guess.

Comment (saikocat) [reply assets/_markdown.scss]: Thank for the advice! I've addressed your comment accordingly.

**Comment by alex-shpak** [comment]
👍

**Pull request merged**